sdcm/utils/k8s/chaos_mesh.py

```diff
@@ -71,14 +71,16 @@ def initialize(self) -> None:
             self._k8s_cluster.POOL_LABEL_NAME, self._k8s_cluster.AUXILIARY_POOL_NAME)
         scylla_node_pool_affinity = get_helm_pool_affinity_values(
             self._k8s_cluster.POOL_LABEL_NAME, self._k8s_cluster.SCYLLA_POOL_NAME)
+        chaos_daemon_settings = scylla_node_pool_affinity | {
+            "runtime": "containerd", "socketPath": "/run/containerd/containerd.sock"}
```

Comment:
is this correct for both EKS and kind ?

Reply:
actually it is not - I added code to detect runtime and adapt accordingly.